journey/pepr-deploy.ts

```diff
@@ -49,6 +49,8 @@ export function peprDeploy() {
     }
 
     it("npx pepr monitor should display validation results to console", async () => {
+      await testValidate();
```

Comment:
Make sure the sample is deployed betfore running the `monitor` command. It is something timing out. `testValidate()` is idempotent and produces the same results every time